app/objective/get.controller.js

```diff
@@ -1,6 +1,6 @@
 const { logger } = require('../../common/logging/logger')
-const { getSentencePlanObjective } = require('../../common/data/sentencePlanningApi')
-const { removeUrlLevels } = require('../../common/utils/util')
+const { getSentencePlanObjective, getSentencePlanNeeds } = require('../../common/data/sentencePlanningApi')
+const { removeUrlLevels, sortObject } = require('../../common/utils/util')
 
 const getObjective = async (
   { path, errors, errorSummary, body, renderInfo, params: { planId, objectiveId }, session: { 'x-auth-token': token } },
```

Comment:
Worth destructing body and setting any required defaults there? 
body: { objective = null, needs = [] }